src/oui/oui-tile/tile.less

```diff
@@ -1,7 +1,116 @@
-@import './_mixins';
 @import './_normalize';
+@import './_variables';
 
 .oui-tile {
   #oui > .tile-normalize();
-  #oui > .tile-base();
+
+  padding: @oui-tile-padding;
+  background-color: @oui-tile-background-color;
+  border: 1px solid @oui-tile-border-color;
+  border-radius: @oui-tile-border-radius;
+  box-shadow: @oui-tile-box-shadow;
+
+  &__title {
+    font-weight: @oui-font-semibold;
+    color: @oui-tile-title-color;
+  }
+
+  &__legend {
+    color: @oui-tile-title-color;
+  }
+
+  &__button {
+    margin: 0;
+    padding-left: @oui-tile-item-side-spacing;
+    padding-right: @oui-tile-item-side-spacing;
+    margin: 0 -@oui-tile-item-side-spacing;
+    box-sizing: content-box;
+    font-weight: @oui-font-semibold;
+    text-align: left;
+    text-decoration: none;
+
+    &.oui-button_full-width.oui-button_icon-right {
+      .oui-icon {
+        &::before {
+          right: @oui-tile-item-side-spacing;
+        }
+      }
+    }
+
+    &:hover, &:focus {
+      background-color: @oui-tile-background-hover;
+    }
+  }
+
+  &__button.oui-button_link {
+    &,
+    &:hover,
+    &:focus { text-decoration: none; }
+  }
+
+  &__item {
+    position: relative;
+    display: flex;
+
+    border-top: 1px solid @oui-tile-item-border-color;
+    padding: @oui-tile-item-top-spacing 0;
+
+    &:last-child {
+      border-bottom: 1px solid @oui-tile-item-border-color;
+    }
+
+    // stylelint-disable max-nesting-depth
```

Comment:
There is no `.stylelintrc` file nor scripts in `package.json` which is running stylelint, so we can get rid of this line I guess.